tests/helpers/acceptance.js

```diff
@@ -55,6 +57,15 @@ function createTestTargets(projectName, options) {
   options.command = options.command || 'new';
 
   return applyCommand(options.command, projectName, '--skip-npm', '--skip-bower', `--directory=${outputDir}`)
+    .then(() => isExperimentEnabled('MODULE_UNIFICATION') && getURLFor('canary'))
```

Comment:
This should be fixed by #8432. Thanks for investigating and reporting the issue.